nf1_cellpainting_env.yml

```diff
@@ -0,0 +1,23 @@
+name: nf1_cellpainting_data
```

Comment:
I can see the benefits of being strict so that an individual would be the same results (e.g., reproducibility), but I also see your point from the discussion that it is much easier for the environment to decay over time. I will add `>:` to the files since these package versions (as far as I know) with the specified versions, and I don't see anytime soon that the versions above this would break anything.

FYI for @roshankern that we had this discussion regarding environments. I think Greg has a point that being more flexible is also very good for the long term but being strict has its benefits. More of a balance game.